interfaces/IBF-dashboard/src/app/components/set-trigger-confirm-popover/set-trigger-confirm-popover.component.scss

```diff
@@ -0,0 +1,3 @@
+ion-label {
```

Comment:
Use tailwind's [text-xs](https://tailwindcss.com/docs/font-size) instead.

Reply:
closing this, as this popover will go